doc/api/net.md

```diff
@@ -1000,7 +1000,7 @@ If `allowHalfOpen` is set to `true`, when the other end of the socket
 sends a FIN packet, the server will only send a FIN packet back when
 [`socket.end()`][] is explicitly called, until then the connection is
 half-closed (non-readable but still writable). See [`'end'`][] event
-and [RFC 1122][half-closed] for more information.
+and [RFC 1122][half-closed] (section 4.2.2.13) for more information.
```

Comment:
Maybe this should stay

Reply:
What exactly should stay? I have not remove here anything, I've just added a note.

Reply:
Read this other way around 🤦‍♂️